test/themes.ts

```diff
@@ -15,8 +15,8 @@ test('Installing a theme', async (t: any) => {
 
   await logIn(t);
 
-  await app.client.waitForExist('.top-nav.loading', 5000, true);
-  await app.client.click('button=Themes');
+  // await app.client.waitForExist('button=themes', 5000, true);
```

Comment:
Seems like it's gonna be flaky

Reply:
how so?

Reply:
Just reviewed the code. We have updated `logIn(t)` method now. So this is not gonna be an issue.